Formula/php.rb

```diff
@@ -70,6 +70,9 @@ def install
     ENV.append "CPPFLAGS", "-DU_USING_ICU_NAMESPACE=1"
 
     config_path = etc/"php/#{php_version}"
+    # Prevent system pear config from inhibitting pear install
```

Comment:
If we're pouring a bottle, isn't this still a concern? Should we handle it in post_install?

Reply:
No, It's only build time. post_install rebuilds it.

Reply:
only one "t" in the word "inhibiting"

Reply:
![](http://www.beeroftomorrow.com/wp-content/uploads/2016/03/wiggum-giphy.gif)